MTM-62651 Added release notes for withChildren default value change (#3157)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-20-25-113-0_MTM-62651_default-value-for-the-inventory-withChildren-parameter.md

---

> date: '2025-04-01'

Reviewer: ```suggestion
date: 
```

title: Inventory API – change of the default value for the withChildren parameter
product_area: Platform services
change_type:

>   - value: change-VSkj2iV9m

Reviewer: ```suggestion
  - value: change-3BQrQ6adS
--

</div></b>
```

>     label: Fix

Reviewer: ```suggestion
    label: API change
```

component:
  - value: component-JlFdtOPva
    label: Rest API
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-62651
version: 2025.113.0
---

> The default value of the `withChildren` parameter for the `GET /inventory/managedObjects` endpoint is changing from `true` to `false`. To continue receiving child assets, devices, and additions in the response, clients must explicitly include `?withChildren=true` in their requests. This change aligns with previous announcements and aims to improve performance and response clarity.

Reviewer: ```suggestion
The default value of the `withChildren` parameter for the `GET /inventory/managedObjects` endpoint has changed from `true` to `false`. To continue receiving child assets, devices, and additions in the response, clients must explicitly include `?withChildren=true` in their requests. This change aligns with previous announcements and aims to improve performance and response clarity.
```

> The default value of the `withChildren` parameter for the `GET /inventory/managedObjects` endpoint has changed from `true` to `false`. To continue receiving child assets, devices, and additions in the response, clients must explicitly include `?withChildren=true` in their requests. This change aligns with previous announcements and aims to improve performance and response clarity. For more details, see the [Announcement](https://drive.google.com/file/d/1vukBH7i_8yT-lLcH41k_zYoVdeFArlTs/view?usp=sharing).

Reviewer: @pkepa-c8y We cannot add this link as this an internal location. Perhaps we can state "with previous announcements (Release 10.18)"... These release notes are provided to customers on request via support.

Author: @BeateRixen I thought so too. Can I find a link to the previous documentation somewhere or is it all packed in a zip like in the link below https://github.com/Cumulocity-IoT//c8y-docs/releases ?

Reviewer: @pkepa-c8y Yes, it is packed in a zip and it is available as PDFs but not publicly. You need to go via support. That is intentional. So you cannot add a link. But you could simply mention that it was announced in the 10.18 release.

Author: @BeateRixen changed.

---
date: 
title: Inventory API – change of the default value for the withChildren parameter
product_area: Platform services
change_type:

>   - value: change-3BQrQ6adS
> --
> 
> 
> </div></b>
>     label: API change

Reviewer 2: This looks malformed, isn't it?
```suggestion
  - value: change-3BQrQ6adS
    label: API change
```
